src/wazuh_modules/vulnerability_detector/wm_vuln_detector_db.h

```diff
@@ -74,6 +74,8 @@ typedef enum vu_query {
     VU_REMOVE_UNUSED_STATES,
     // WAZUH DB REQUESTS
     VU_HOTFIXES_REQUEST,
+    VU_HOTFIXES_GET,
+    VU_SOFTWARE_GET,
```

Comment:
I think this query could now be named VU_PACKAGES_GET to be aligned with the commands names.

Reply:
Fixed